app/models/label.rb

```diff
@@ -59,7 +60,23 @@ def choose_label_color
     c33c54 254e70 37718e 8ee3ef aef3e7
   ].freeze
 
+  def to_h
+    {
+      id:,
+      name:,
+      color:,
+      font_color:,
+      used_by_tasks: tasks.loaded? ? tasks.size : 0,
+      created_at: created_at.to_fs(:rfc822),
+      updated_at: updated_at.to_fs(:rfc822),
+    }
```

Comment:
another way would be to use
```
{
...
used_by_tasks: (tasks.size if tasks.loaded?),
...
}.compact
```
This way the used_by_tasks key would not be in there at all which is more explicit than just silently setting used_by_tasks to 0. What do you think?

Reply:
This sounds even better, actually. I also thought about returning `nil` rather than `0`, but omitting the key is even smarter.